attestation-service/verifier/src/tdx/mod.rs

```diff
@@ -31,56 +32,52 @@ pub struct Tdx {}
 impl Verifier for Tdx {
     async fn evaluate(
         &self,
-        nonce: String,
-        attestation: &Attestation,
+        evidence: &[u8],
+        report_data: Option<&[u8]>,
+        init_data_hash: Option<&[u8]>,
```

Comment:
Hi @jialez0 , I've changed the verifier API of TDX. Please take a look if it looks good to you.